src/js/components/CheckBox/checkbox.stories.js

```diff
@@ -194,6 +194,55 @@ class CheckBoxInsideButton extends Component {
   }
 }
 
+class CheckBoxWithStickyDiv extends Component {
+  state = {
+    checked: [],
+    checkboxes: Array(8)
+      .fill()
+      .map((_, i) => `item ${i + 1}`),
+  };
+
+  onCheck = (event, value) => {
+    const { checked } = this.state;
+    if (event.target.checked) {
+      checked.push(value);
+      this.setState({ checked });
+    } else {
+      this.setState({ checked: checked.filter(item => item !== value) });
+    }
+  };
+
+  render() {
+    const { checked, checkboxes } = this.state;
+    return (
+      <Grommet theme={grommet}>
+        <Box align="center" pad="large">
+          <div
+            style={{
+              height: '100px',
+              width: '150px',
+              overflow: 'auto',
+              position: 'relative',
+            }}
+          >
+            <div style={{ background: '#EEE', position: 'sticky', top: 0 }}>
+              Click Me!
+            </div>
+            {checkboxes.map(item => (
+              <CheckBox
+                key={item}
+                checked={checked.indexOf(item) !== -1}
+                label={item}
+                onChange={e => this.onCheck(e, item)}
+              />
+            ))}
```

Comment:
Thanks for adding a story to show this!
Would you mind avoiding using the style prop where applicable box props are available? (height, width, overflow)
And maybe convert the div into a box as well?
I am guessing a lot of people are c/p from storybook so we should make sure we promote "best practices"

Reply:
@oorestisime Thanks for reviewing. That's exactly what I thought before pushing this up, but I wanted to kind of show/address specifically the issue with sticky div. I will update the fix.

Reply:
Updated the div's with Box.